GetAssetTrigger/index.ts

```diff
@@ -0,0 +1,74 @@
+import { AzureFunction, Context, HttpRequest } from "@azure/functions"
+import axios, { AxiosError } from "axios"
+import type { Asset } from "@equinor/data-marketplace-models"
+import { config } from "../config"
+
+const GetAssetTrigger: AzureFunction = async function (context: Context, req: HttpRequest): Promise<void> {
+  const { id } = context.bindingData
+
+  try {
```

Comment:
We should schedule this long awaited from try/catch to functional ninja workshop soon @nilsml

Reply:
probably 😅